lib/galaxy/web/base/interactive_environments.py

```diff
@@ -162,16 +162,19 @@ def docker_cmd(self, env_override={}):
         conf = self.get_conf_dict()
         conf.update(env_override)
         env_str = ' '.join(['-e "%s=%s"' % (key.upper(), item) for key, item in conf.items()])
-        return '%s run %s -d %s -p %s:%s -v "%s:/import/" %s' % \
+        volume_str = ' '.join(['-v "%s"' % volume for volume in volumes])
```

Comment:
Hi @jmchilton,

thanks for looking over it. This looks like a nice idea.
Do you mean something like the following?
https://github.com/bgruening/galaxy/commit/bc9f707c0ce2619021cdc3e3d4f67b59e4824b14
and the corresponding make changes here:
https://github.com/bgruening/galaxy/commit/63d475e4687f778405a5b1f94e4687c51264ade2